src/app/teacher/edit-tag-dialog/edit-tag-dialog.component.html

```diff
@@ -0,0 +1,15 @@
+<h2 mat-dialog-title i18n>Edit Tag</h2>
+<mat-dialog-content>
+  <mat-form-field class="tag-name">
+    <mat-label i18n>Tag Name</mat-label>
+    <input matInput (keypress)="keyPressed($event)" [formControl]="tagControl" autofocus />
```

Comment:
Call save() directly on keyup.enter event?
```suggestion
    <input matInput (keyup.enter)="save()" [formControl]="tagControl" autofocus />
```